server/routes/overview/overviewControllerV2.ts

```diff
@@ -30,7 +30,12 @@ export default class OverviewControllerV2 {
       const { lastUpdatedBy, lastUpdatedDate, lastUpdatedAtPrisonName, noGoals } =
         this.getLastUpdatedGoalData(prisonerGoals)
 
-      const goalCounts = this.calculateGoalCounts(prisonerGoals.goals)
+      let goalCounts
+      if (prisonerGoals.problemRetrievingData) {
```

Comment:
If there was a problem retrieving the data why do we need the goal counts? 🤔 
Does the nunjucks template still show the tab headings as well as the message re: problem retrieving data? 🤔